### ISSUE TITLE
xhtml2pdf hangs forever on simple input

### ISSUE THREAD

**Issue by MirkoDziadzka** [xhtml2pdf/xhtml2pdf#209]
The following minimal HTML page let xhtml2pdf hangs forever (ok, I stopped it after 20 seconds or so)

```
<!DOCTYPE html>
<html> <head> <title>Test</title>
<style> @page { #foo { } } </style>
</head><body></body> </html>
```
Save this to a file ```foo.html``` and run  ```python -m xhtml2pdf.pisa foo.html```

please note that this is probably wrong syntax, but it should not hang.

**Comment by benjaoming** [comment]
Confirmed, let's get that out of the way.

**Comment by benjaoming** [comment]
Fixed in #284
